### PR TITLE
adding back button to delete popup

### PR DESCRIPTION
**Pull request by fazlan-nazeem**

### CODE REVIEW

File: apps/ml/site/data/datasets.jag [hunk 1]
```diff
@@ -102,6 +103,13 @@ include("../../includes/tenantAware.jag");
 
             <!-- delete dataset popup -->
             <div class="popup-content" data-operation="delete-dataset">
+            	<div  id="back-delete-dataset" align="left">
```
Comment (cdathuraliya) [inline apps/ml/site/data/datasets.jag:106]: Please format.

File: apps/ml/site/data/datasets.jag [hunk 2]
```diff
@@ -112,6 +120,13 @@ include("../../includes/tenantAware.jag");
 
             <!-- delete dataset version popup -->
             <div class="popup-content" data-operation="delete-dataset-version">
+           		 <div  id="back-delete-dataset-version" align="left">
```
Comment (cdathuraliya) [inline apps/ml/site/data/datasets.jag:123]: Please format.

File: apps/ml/site/data/datasets.jag (hunk 1 above)
Comment (fazlan-nazeem) [inline apps/ml/site/data/datasets.jag:106]: is it the space between div and id?

File: apps/ml/site/data/datasets.jag (hunk 2 above)
Comment (fazlan-nazeem) [inline apps/ml/site/data/datasets.jag:123]: fixed with https://github.com/wso2/carbon-ml/pull/200

File: apps/ml/site/data/datasets.jag (hunk 1 above)
Comment (fazlan-nazeem) [inline apps/ml/site/data/datasets.jag:106]: fixed with https://github.com/wso2/carbon-ml/pull/200

File: apps/ml/site/project/projects.jag [hunk 3]
```diff
@@ -344,6 +359,12 @@ include("../../includes/tenantAware.jag");
             window.location.href = 'compare.jag?projectId=' + projectId + '&projectName=' + projectName + '&dataseId=' + dataseId;
         });
 
+        $(document).on('click','#back-delete-project, #back-delete-analysis',function(e){
+        	e.preventDefault();
+
```
Comment (prabathabey) [inline apps/ml/site/project/projects.jag:364]: Remove unnecessary newline.

Comment (fazlan-nazeem) [inline apps/ml/site/project/projects.jag:364]: Fixed with PR https://github.com/wso2/carbon-ml/pull/201

File: apps/ml/site/project/compare.jag [hunk 4]
```diff
@@ -643,7 +643,14 @@
                 '</div>';
         };
         return accordionPanels;
-    }    
+    }  
+
+     $(document).on('click','#back-download-model, #back-publish-model',function(e){
+            e.preventDefault();
+
```
Comment (sinthuja) [inline apps/ml/site/project/compare.jag:650]: Remove unnecessary new lines.

Comment (fazlan-nazeem) [inline apps/ml/site/project/compare.jag:650]: Fixed with PR https://github.com/wso2/carbon-ml/pull/202

**Pull request merged**